http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSAlgebra.scala

```diff
@@ -29,3 +29,22 @@ final case class DeleteAKSAppParams(
   cloudContext: AzureCloudContext,
   keepHistory: Boolean = false
 )
+
+/** Enumerates the possible identity modes for an AKS app. */
+sealed trait IdentityType
+object IdentityType {
```

Comment:
Trying to keep track of the different "identity modes" currently at play. I think eventually we'll want to move toward everything using Workload Identity.